lib/rubocop/cop/rails/validation.rb

```diff
@@ -48,6 +48,24 @@ def on_send(node)
         def preferred_method(method)
           WHITELIST[BLACKLIST.index(method.to_sym)]
         end
+
+        def autocorrect(node)
+          _receiver, method_name, *args = *node
+          options = args.delete_if { |arg| arg.type == :sym }.first
+          lambda do |corrector|
+            validate_parameter = method_name.to_s.split('_')[1]
+            corrector.replace(node.loc.selector, 'validates')
+            if options
+              corrector.replace(
+                options.loc.expression,
+                "#{validate_parameter}: { #{options.source} }")
```

Comment:
I mean `autocorrect_source` is the [method](https://github.com/bbatsov/rubocop/blob/49598a035cd39a75ec91466a4e1a79c7e4f0f5f9/spec/support/cop_helper.rb#L42)